psi4/src/psi4/libmoinfo/model_space.h

```diff
@@ -35,11 +35,9 @@
 */
 
 #include "slater_determinant.h"
+#include "moinfo.h"
 
 namespace psi {
-
-class MOInfo;
```

Comment:
I understood it to be better (faster compile) to forward declare the class like this rather than including the header. Am I missing something here?

Reply:
Hmm, I have not thought of that to be honest. I made the change because I felt like it makes the code more readable, easier to see what a particular file depends on. But of course, I can revert if you feel strongly about the potential compile time impact of it.

Reply:
I'd favor reverting to the forward decl. But no hurry if you want to wait until FAE chimes in.

Reply:
I think forward declaration is better because it avoids including (expanding the content of the .h file) in all the files that also include `model_space.h`.

Reply:
Fair point, reverted.